discarding /cm/shared/openmind/miniconda/3.18.9-py3/bin from PATH
prepending /home/gretatu/.conda/envs/brainmodeling/bin to PATH
Traceback (most recent call last):
  File "get_activations.py", line 192, in <module>
    model = GPT2(args.model, sentence_embedding=args.sentence_embedding)
  File "get_activations.py", line 32, in __init__
    self.model = GPT2Model.from_pretrained(model_instance, output_hidden_states=True, return_dict=True) #return_dict=True returns output as dict and not plain tuple.
  File "/home/gretatu/.conda/envs/brainmodeling/lib/python3.6/site-packages/transformers/modeling_utils.py", line 655, in from_pretrained
    model = cls(config, *model_args, **model_kwargs)
TypeError: __init__() got an unexpected keyword argument 'return_dict'
